### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by Ibochkarev** [avocode/avocode#402]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.4.2-d618096
Source UID: 84484
```
TypeError: Cannot read property 'x' of null
  at e.calculateBounds (/Users/ibochkarev/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/select-tool/lib/select-tool.js:1:8095)
  at e.mousemoveDrag (/Users/ibochkarev/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/select-tool/lib/select-tool.js:1:4971)
  at /Users/ibochkarev/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/select-tool/lib/select-tool.js:1:240
  at t.module.exports.Emitter.emit (/Users/ibochkarev/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/emissary/lib/emitter.js:118:15)
  at t.mousewheel (/Users/ibochkarev/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/stage.js:1:13091)
  at HTMLCanvasElement.<anonymous> (/Users/ibochkarev/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/stage.js:1:438)

```

**Comment by petrbrzek** [comment]
Do you know how did it happen?

**Comment by petrbrzek** [comment]
Hello, we’ve just released Avocode 2.4.3 that should fix the issue. Let us know if the problem persists.

**Comment by Ibochkarev** [comment]
Thanks!
